code/API_definitions/location.yaml

```diff
@@ -181,6 +184,12 @@ components:
       minimum: 2
       maximum: 200
       example: 50
+    MaxAge:
+      description: Max location aging validity required by the application in Minute, Default =0 for realtime location
```

Comment:
We were discussing in the original PR whether this new parameter should be mandatory or not. If mandatory, it makes no sense to define or document a default value.